### DISCUSSION TITLE
Windows - getting syntax hightlighting and hints working

### DISCUSSION THREAD

**Discussion by rpherman** [helix-editor/helix#5217] [General] [answered]
I have built Helix from source for my Windows laptop, and it works great, however, I have not been able to get syntax highlighting to work. I've tried for Rust and Python and C so far. I have made sure rust-analyzer.exe symlink is in my root helix directory, but after swtiching lang to rust and opening a *.rs file, I do not get syntax highlighting. Please is there a more detailed setup/install for Windows? I am not sure where things go in the directory structure or how to setup LSP and Treesitter.

**Comment by kirawi** [comment, marked as answer]
You need to link the runtime folder: https://docs.helix-editor.com/install.html#build-from-source

**Reply by rpherman** [reply]
I had already linked it to the runtime folder in the helix root folder - helix/runtime. There is also one in the helix/helix-term/$PWD folder.

The solution was to run the two commands:

`hx --grammar fetch; hx --grammar build`

Thanks!
